model_analysis/imagenet_id.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np 
```

Comment:
Add header comment explaining the use of this file. How is this different from `~/inference/imagenet_id.py`?